packages/shell-api/src/database.ts

```diff
@@ -1158,4 +1149,15 @@ export default class Database extends ShellApiClass {
   async printSlaveReplicationInfo(): Promise<CommandResult> {
     throw new MongoshInvalidInputError('Method deprecated, use db.printSecondaryReplicationInfo instead');
   }
+
+  @serverVersions(['3.1.0', ServerVersions.latest])
+  watch(pipeline: Document[] = [], options: ChangeStreamOptions = {}): ChangeStreamCursor {
+    this._emitDatabaseApiCall('watch', { pipeline, options });
+    const cursor = new ChangeStreamCursor(
+      this._mongo._serviceProvider.watch(pipeline, options, {}, this._name),
+      this[asPrintable]()
```

Comment:
ditto here, 
```suggestion
      this._name
```

?

(Fwiw, it stumped me a bit that we print Databases and Collections as if they were plain strings when I first started using mongosh … I don’t necessarily have a better suggestion, though :))

Reply:
Yeah it's a bit odd, we can revisit in the shell 2.0 :D